workshop/content/030_basic_content/060_advanced_experiments/030_modify_tags_security/_index.en.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Tags: update vs. create"
+weight: 30
+services: false
+draft: false
+---
+
+{{% notice note %}}
+This section is aimed at large, distributed, and _extremely_ security conscious teams. If that's not a high concern to you, feel free to skip this section.
+{{% /notice %}}
+
+As we saw in the previous secion, tags can be used as part of access control policies. Because tags and experiment templates are managed by independent services it is possible to atomically _create_ an experiment template with associated tags but it is not possible to atomically _update_ tags and experiment template content.
```

Comment:
secion > section

Reply:
Because tags and experiment templates are managed by independent services it is possible to atomically _create_ an experiment template with associated tags but it is not possible to atomically _update_ tags and experiment template content. 

>>>  I would change to.

Because tags and experiment templates are managed by independent services it is not possible to atomically  _update_ tags and experiment template content.

Reply:
Fixed section typo and simplified explanation